astro/src/content/docs/apis/identity-providers/_link-response-body-base.mdx

```diff
@@ -15,31 +15,31 @@ import InlineField from 'src/components/InlineField.astro';
   <APIField name={ props.base_field_name + ".identityProviderId" } type="UUID">
     The unique Id of the identity provider.
   </APIField>
-  <APIField name={ props.base_field_name + ".identityProviderType" } type="String" since="1.47.0">
+  <APIField name={ props.base_field_name + ".identityProviderName" } type="String" since="1.47.0">
     The name of the identity provider.
   </APIField>
-  <APIField name={ props.base_field_name + ".identityProviderUserId" } type="String">
+  <APIField name={ props.base_field_name + ".identityProviderType" } type="String" since="1.47.0">
```

Comment:
Type was added in 1.46.0.
- https://github.com/FusionAuth/fusionauth-issues/issues/2218

Reply:
```suggestion
  <APIField name={ props.base_field_name + ".identityProviderType" } type="String" since="1.46.0">
```